molecule/default/tests/test_default.py

```diff
@@ -52,6 +85,24 @@ def test_services(host, svc):
 
 
 @pytest.mark.parametrize("command", ["docker-compose version"])
```

Comment:
```suggestion
```

Reply:
Please see commit fb527a4.

Reply:
Ah, sorry for forgetting to adjust the function signature.